queries.sql

```diff
@@ -40,3 +40,25 @@ SELECT COUNT (*) FROM animals WHERE neutered = false;
 SELECT AVG(wieght_kg) FROM animals;
 SELECT MAX(escape_attempts) FROM animals;
 SELECT species, MAX(wieght_kg) FROM animals GROUP BY species;
+
+/* Day 3 Project */
+SELECT name FROM animals A 
+JOIN owners O ON O.id = A.owner_id AND O.full_name = 'Melody Pond';
+
+SELECT * FROM animals A
+JOIN species S ON A.species_id = S.id AND A.species_id = 1;
+
```

Comment:
- [ ] Instead of adding a condition clause for the `species_id` to equal `1`, kindly use the name of the species instead of its `id`. As per [this](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=List%20of%20all%20animals%20that%20are%20pokemon%20(their%20type%20is%20Pokemon).) requirement, you should use the name of the species and not its `id`.